src/frontend/src/pages/FinancePage/FinanceComponents/ReportRefundModal.tsx

```diff
@@ -72,13 +75,30 @@ const ReportRefundModal: React.FC<ReportRefundProps> = ({ modalShow, handleClose
         <LoadingIndicator />
       ) : (
         <FormControl>
+          <FormLabel>Amount</FormLabel>
           <ReactHookTextField
             name="refundAmount"
             control={control}
             sx={{ width: 1 }}
             startAdornment={<AttachMoneyIcon />}
             errorMessage={errors.refundAmount}
           />
+
+          <FormLabel sx={{ paddingTop: 2 }}>Date Received</FormLabel>
```

Comment:
I'm not too sure if this or wrapping each field in `FormControl` is the right way to add spacing between spacing between fields

Reply:
this looks good